### DISCUSSION TITLE
API: Create Project

### DISCUSSION THREAD

**Discussion by CodyPChristian** [spatie/flareapp.io-roadmap#68] [Ideas]
It would be very useful to be able to create projects via API, I find everything else I'd need is there but in my case I need to deploy flare to approx 40 laravel apps and more can be created at any moment so I need to add flare into our install pipeline.

**Comment by rubenvanassche** [comment]
Implemented: https://flareapp.io/api-docs#tag/projects/paths/~1projects/post

**Reply by CodyPChristian** [reply]
Awesome thank you, sadly just finally saw this in my notifications.
